packages/cip2/src/selectionConstraints.ts

```diff
@@ -10,6 +10,8 @@ import {
 } from './types';
 import { ProtocolParametersRequiredByInputSelection } from '.';
 
+const coinsPerUTxOByteToCoinsPerUTxOWord = (coinsPerByte: number): number => coinsPerByte * 8;
```

Comment:
@AngelCastilloB what is the source for `*8` :thinking:? I'm not confident that's correct, because:
Current value of this parameter in testnet is `4310*8=34480`
Old value (perWord) was `34482`, which is slightly higher than that.

However, when running one of our e2e tests (using old param) that had 1M coins in an output I got a tx submission error, which doesn't make sense as minimum coin should have become lower instead of higher :thinking:

Reply:
Ok, let's do a `*9` then so that we at least get enough coin?

Reply:
@mkazlauskas Yes the multiplication by 8 is an approximation (the conversion is done the same way by the ledger team see [here](https://github.com/input-output-hk/cardano-ledger/blob/master/eras/babbage/impl/src/Cardano/Ledger/Babbage/Translation.hs#L191)).

It doesn't yields the same value because the old value was not a multiple of 8, for now this is the only way we have to approximate the old value, however, as @rhyslbw mentioned, this is temporary until we update de CLS that supports the new calculations which require this new coinsPerByte value.

What is the test that is failing? I can take a look :).